configure.ac

```diff
@@ -634,6 +651,28 @@ if test "$cross_compiling" = yes; then
 	*-*-cygwin*)
 		_host_ident=
 		;;
+	*-apple-ios*-simulator)
+		_host_os=`echo $host | cut -d '-' -f3`
+		IOS_DEPLOYMENT_TARGET=${_host_os:3}
```

Comment:
@mhsmith As you may know, `IOS_DEPLOYMENT_TARGET` (and `MACOSX_DEPLOYMENT_TARGET`) has a very specific meaning and purpose with the Apple-supplied build tools in Xcode and the Command Line Tools. It would be good to note that here.

Reply:
I've added inline comments to clarify the role of IOS_DEPLOYMENT_TARGET, and clarified how to specify the deployment target with the `--host` argument.

As for the default value and _PYTHON_HOST_PLATFORM - there's two possibilities:
* `--host=arm64-apple-ios-simulator`
  * `_host_os=ios` 
  * Initial `IOS_DEPLOYMENT_TARGET="" (split at index 3)
  * `_host_ident="-iphoneos-arm64"`
  * `_PYTHON_HOST_PLATFORM=ios-iphoneos-arm64
  * Around L850, `IOS_DEPLOYMENT_TARGET` falls back to the default "12.0"
* `--host=arm64-apple-ios15.4-simulator`
  * `_host_os=ios15.4` 
  * Initial `IOS_DEPLOYMENT_TARGET="15.4"`
  * `_host_ident="15.4-iphoneos-arm64"`
  * `_PYTHON_HOST_PLATFORM=ios15.4-iphoneos-arm64
  * Around L850, `IOS_DEPLOYMENT_TARGET` is used as-is.

Reply:
> _PYTHON_HOST_PLATFORM=ios-iphoneos-arm64

That was my question: shouldn't this also fall back to 12.0? As it stands, it doesn't follow what we've defined as the valid format for `sysconfig.get_platform`. Maybe it doesn't matter at the moment, but that could change in the future.

Reply:
Ah - good catch. I'm guessing I've just never used the fallback in any of my testing/local builds. I've cleaned up this section of the code to remove some duplication, ensure that the 12.0 prefix is included, and the ordering of CPU and ABI is consistent.

Reply:
When running within crossenv, `_PYTHON_HOST_PLATFORM` determines `sysconfig.get_platform`, which determines the wheel tag. So reordering the CPU and ABI here would break compatibility with our entire existing repository of wheels. That doesn't seem worth it for consistency with the non-public `sys.implementation._multiarch`, especially since the two values aren't consistent on most of the other platforms anyway.

Reply:
While compatibility with existing wheels would be a nice touch, I don't think it's especially compelling as a technical argument. There aren't *any* 3.13 wheels yet, so compatibility with *existing* wheels isn't really a consideration; and I'd consider avoiding an inconsistency with existing configurations more important than compatibility with an unofficial legacy.

Reply:
OK, I see from the [BeeWare patch](https://github.com/beeware/Python-Apple-support/blob/3.13-b0/patch/Python/Python.patch#L424) that the iOS branch of `sysconfig.get_platform` is currently implemented in terms of `sys.implementation._multiarch`, so keeping them consistent would be easier.

pip and other installers can be agnostic about this by just treating anything after `ios-12.0-` as an opaque string. So the inconvenience of this change would only fall on Briefcase and mobile-forge. EIther they would have to keep track of a different tag format for Python 3.12 and older, or we'd have to rebuild all the existing wheels with the new format. The latter is probably easier in the long run.